src/getESLint.js

```diff
@@ -63,7 +62,6 @@ export function loadESLintThreaded(key, poolSize, options) {
 
   const local = loadESLint(options);
 
-  /** @type {Worker?} */
   // prettier-ignore
   let worker = (/** @type {Worker} */ new JestWorker(source, workerOptions));
```

Comment:
Here the ts problem - it should be `/** @type {Worker} */ (new JestWorker(source, workerOptions))`

Reply:
@alexander-akait fixed